### DISCUSSION TITLE
Data model

### DISCUSSION THREAD

**Discussion by stigbd** [langrenn-sprint/issue-tracker#8] [Ideas]
Vi bør drodle oss fram til ein datamodell før vi setter i gang med å identifisere potensielle tjenester.

Forslag:
- `Event`: et arrangement som har ein dato og lokasjon. Må inneholde minst en konkurranse, men vil vanligvis inneholde mange konkurranser (`races`). Egenskaper:
- - Deltagere: 
- - Løpsklasser: 
- - Køreplan: konkurranser lagt ut i tid. Har starttid pr race.
- - Dato:
- - Sted: 
- `Race`: ei konkurranse for eit avgrensa sett av deltakarar  (feks delt inn etter alder/kjønn etc).  Egenskaper:
- - Startliste: ordnet liste med deltagere etter starttid
- - Resultatliste: ordnet liste med deltagere etter slutt-tid/rekkefølge i målpassering
- - Mellomtider: midlertidig resultatliste ved et gitt passeringspunkt
- `Heat`: ei spesialisering av `Race`.

**Comment by heming-langrenn** [comment]
Added item  Langrenn Entities
